src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:16886/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:16886/project/issues?id=sample&branch=feature1&issues=AWFXe_C2o_XV5PUaMKmA&open=AWFXe_C2o_XV5PUaMKmA)

[ISSUE_KEY]: AWFXe_C2o_XV5PUaMKmA